command/server.go

```diff
@@ -1141,6 +1141,30 @@ func (c *ServerCommand) Run(args []string) int {
 		}
 	}
 
+	isMlockSet := func() bool {
+		// DisableMlock key has been found and thus explicitly set
+		return strutil.StrListContainsCaseInsensitive(config.SharedConfig.FoundKeys, "DisableMlock") ||
+			// mlock is disabled and hence has been explicitly set
+			config.SharedConfig.DisableMlock ||
+			// envvar set
+			envMlock != ""
+	}
+
+	// ensure that the DisableMlock key is explicitly set if using integrated storage
+	if config.Storage != nil && config.Storage.Type == storageTypeRaft && !isMlockSet() {
+
+		c.UI.Error(wrapAtLength(
+			"ERROR: disable_mlock must be configured 'true' or 'false': Mlock " +
```

Comment:
Is the message people will see when upgrading to 1.20 if they didn't previously have disable_mlock configured? Could you add what the previous default was, so they don't have to go to the docs to look it up in order to get Vault running?

Reply:
Right. This is what they will see on startup. I was debating about adding the callout for 1.20. I'll add it in.